woocommerce/payment-gateway/assets/images/card-cc-invalid.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
this SVG should be [optimized](https://github.com/svg/svgo), or manually by removing the title, desc, defs, and trimming whitespace/newline.